catalog/ui/search-ui/standard/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -63,16 +63,56 @@
         <AD description="Use an external authentication point, such as IdP"
             name="Use External Authentication" id="externalAuthentication"
             required="false" type="Boolean" default="false"
-                />
+        />
 
         <AD name="Type Name Mapping" id="typeNameMapping"
             required="false" type="String" cardinality="10000"
             description="Mapping of display names to content types in the form name=type."
-                />
-	    </OCD>
+        />
+    </OCD>
 
-	<Designate pid="org.codice.ddf.ui.search.standard.properties">
-		<Object ocdref="org.codice.ddf.ui.search.standard.properties"/>
-	</Designate>
+    <OCD description="Workspace Query Monitor"
```

Comment:
This should be in the catalog-ui-search metatype.  Not the standard ui metatype.